### PR TITLE
Add loading of submitted applications to Governments

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add new flags to control which actions can be taken on the inbox page to applications
* Add loading of submitted to LG applications to attached accounts
* Add new Submitted to L/FNG Status

### CODE REVIEW

File: services/apps/portal/src/providers/typeorm/migrations/1673042070216-update_app_statuses.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updateAppStatuses1673042070216 implements MigrationInterface {
+  name = 'updateAppStatuses1673042070216';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `UPDATE "portal"."application_status" SET "label" = 'Submitted to ALC' WHERE "code" = 'SUBM'`,
```
Comment (mhuseinov) [inline services/apps/portal/src/providers/typeorm/migrations/1673042070216-update_app_statuses.ts:8]: I wonder if this executed successfully without ';'

File: services/apps/portal/src/application/application-document/application-document.controller.ts [hunk 2]
```diff
@@ -43,6 +46,9 @@ export class ApplicationDocumentController {
     @Param('documentType') documentType: string,
     @Req() req,
   ): Promise<ApplicationDocumentDto> {
+    //Verify Access
+    await this.applicationService.getIfCreator(fileNumber, req.user.entity);
```
Comment (mhuseinov) [inline services/apps/portal/src/application/application-document/application-document.controller.ts:50]: You could create a wrapper method 'verifyAccess' for 'getIfCreator' that does not return any value and simple calls the 'getIfCreator'. This way you do not need to add a comment on every call

File: services/apps/portal/src/providers/typeorm/migrations/1673042070216-update_app_statuses.ts (hunk 1 above)
Comment (dhaselhan) [reply services/apps/portal/src/providers/typeorm/migrations/1673042070216-update_app_statuses.ts:8]: I believe so, only need ; if you are using multiple statements, if its just a single statement Postgres doesn't seem to care

File: services/apps/portal/src/application/application-document/application-document.controller.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/portal/src/application/application-document/application-document.controller.ts]: Good idea, done!

**Review by mhuseinov — APPROVED** [review]

**Pull request merged**